src/utils/testHelpers.ts

```diff
@@ -82,9 +82,16 @@ export function mockAPICalls(testName: string, url = MainnetUrl) {
  * or the one with attached INFURA SECRET key from
  * UNSTOPPABLE_RESOLUTION_INFURA_PROJECTID env variable if any
  */
-export function secretInfuraLink(): string {
+export function secretInfuraLink(option: InfuraProtocolOptions = InfuraProtocolOptions.http): string {
```

Comment:
`option` - let's make this name more self-descriptive, like `infuraProtocol`.
Also I don't fancy `Options` prefix in the name of the enum type, it seems to be a bit irrelevant for enums